### PR TITLE
Adding new file to branch about my day

### PR DESCRIPTION
**Pull request by NatashaChernyshova**

### CODE REVIEW

File: nata2 (2).txt [hunk 1]
```diff
@@ -0,0 +1 @@
+monday
```
Comment (LilyWTAUA) [inline nata2 (2).txt:1]: thanks

**Review by YanaKucher — APPROVED** [review]
ok

**Review by SashaMyryniuk — APPROVED** [review]
happy monday!

**Review by LilyWTAUA — COMMENTED** [review]
ok)

**Pull request merged**